docs/podman-run.1.md

```diff
@@ -1129,6 +1129,15 @@ KillMode=process
 WantedBy=multi-user.target
 ```
 
+### Configuring Storage Options from the command line
+
+Podman allows for the configuration of storage by changing the values
+in the /etc/container/storage.conf or by using global options.  This
+shows how to setup and use fuse-overlayfs for a one time run of busybox
+using global options.
+
+podman --log-level=debug --storage-driver overlay --storage-opt "overlay.mount_program=/home/podman/podman/usr/local/bin/fuse-overlayfs" run busybox /bin/sh
```

Comment:
The default path for fuse-overlayfs is `/usr/bin/fuse-overlayfs` - we might want to use that

Reply:
changed